notebooks/Notebook1.md

```diff
@@ -9,6 +9,12 @@ Did it help you make a design decision? Was it helpful in addressing the most pr
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+The critiques I received in class last week helped me better pin down what the goals of my project are.
+First and foremost, I want a versatile language that is able to describe a large variety of games. This means that making my language overly restrictive in any way will fundamentally limit the types of games that can be created; thus, the language should still allow for many general-purpose programming language concepts. While this might mean that the language errs close to the border of DSL and library, it is critical that the language remains as flexible as possible.
```

Comment:
I think this is a good goal, though perhaps the way to achieve it would be to work from a small set of features towards something that is more general-purpose.

Was there some specific feedback from your critique partner(s) that made you want to push the language towards something that is more general-purpose?